lib/bitex_bot/models/api_wrappers/itbit/itbit_api_wrapper.rb

```diff
@@ -28,15 +28,15 @@ def self.find_lost(type, price, _quantity)
   end
 
   def self.order_book
-    order_book_parser(Itbit::XBTUSDMarketData.orders)
+    order_book_parser(market.orders)
   end
 
   def self.orders
-    Itbit::Order.all(status: :open).map { |o| order_parser(o) }
+    Itbit::Order.all(instrument: currency_pair, status: :open).map { |o| order_parser(o) }
   end
 
   def self.place_order(type, price, quantity)
-    Itbit::Order.create!(type, :xbtusd, quantity.round(4), price.round(2), wait: true)
+    Itbit::Order.create!(type, currency_pair, quantity.round(4), price.round(2), wait: true, currency: currency_base)
```

Comment:
As per ItBit's docs, it's not clear what the `currency` argument is used for and the wrapper was already working without it.
"If it ain't broke, don't fix it."